src/structures/Utils.ts

```diff
@@ -163,41 +176,67 @@ export abstract class TrackUtils {
   static async getClosestTrack(
     unresolvedTrack: UnresolvedTrack
   ): Promise<Track> {
-    if (!TrackUtils.manager) throw new RangeError("Manager has not been initiated.");
+    if (!TrackUtils.manager)
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=7585532)
```suggestion
    if (!TrackUtils.manager)
```